RELEASE_NOTES.md

```diff
@@ -48,6 +46,22 @@ Kitchen-ec2 has been updated to 3.0, which uses the newer `aws-sdk-v3` and inclu
 
 Chef Provisioning is no longer included with Chef DK, and will be officially end of life on August 31, 2019. The source code of Chef Provisioning and the drivers have been moved into the chef-boneyard GitHub organization and will not be further maintained. Current users of Chef Provisioning should contact your Chef Customer Success Manager or Account Representative to review your options.
 
+### `knife bootstrap` against cloud providers
+
+`knife bootstrap` was
```

Comment:
This paragraph has some funky line breaks, which I would recommend being fixed. (Maybe a sentence per line?) This is not a must, however.

Reply:
Broke the lines up to 1-per-sentence for readability